routes/transaction.go

```diff
@@ -804,16 +779,48 @@ func (fes *APIServer) BurnBitcoinStateless(ww http.ResponseWriter, req *http.Req
 
 		SerializedTxnHex:   hex.EncodeToString(bitcoinTxnBytes),
 		TxnHashHex:         bitcoinTxn.TxHash().String(),
-		BitCloutTxnHashHex: bitCloutTxnHashHex,
+		BitCloutTxnHashHex: bitcloutTxnHashString,
 
 		UnsignedHashes: unsignedHashes,
 	}
-	if err := json.NewEncoder(ww).Encode(res); err != nil {
+	if err = json.NewEncoder(ww).Encode(res); err != nil {
 		_AddBadRequestError(ww, fmt.Sprintf("BurnBitcoin: Problem encoding response as JSON: %v", err))
 		return
 	}
 }
 
+// GetNanosFromSats - convert Satoshis to BitClout nanos
+func (fes *APIServer) GetNanosFromSats(satoshis uint64, feeBasisPoints uint64) (uint64, error){
+	usdToBTC, err := GetUSDToBTCPrice()
+	if err != nil {
+		return 0, fmt.Errorf(" Problem getting usd to btc exchange rate: %v", err)
+	}
+	usdCentsPerBitcoin := usdToBTC * 100
+	usdCents := (float64(satoshis) * usdCentsPerBitcoin) / math.Pow(10, 8)
```

Comment:
would be great if we could store `math.Pow(10, 8)` as a constant somewhere (maybe we already have it?)